src/components/sortingFunctions.js

```diff
@@ -0,0 +1,42 @@
+export const getShoppingItemBackgroundStyles = (
+  daysLeftForNextPurchase,
+  getDaysBetweenCurrentAndPreviousPurchase,
+  lastPurchasedOn,
+) => {
+  const presentDate = Date.now();
+  const daysBetweenCurrentAndPreviousPurchase = getDaysBetweenCurrentAndPreviousPurchase(
+    lastPurchasedOn,
+    presentDate,
+  );
+  if (daysBetweenCurrentAndPreviousPurchase >= daysLeftForNextPurchase * 2) {
+    return 'grey';
+  } else if (daysLeftForNextPurchase <= 7) {
+    return 'green';
+  } else if (daysLeftForNextPurchase > 7 && daysLeftForNextPurchase < 30) {
```

Comment:
Here you can write just`daysLeftForNextPurchase < 30` if you want because in order to reach the second `else-if` it means that the condition in the first `else-if` wasn't satisfied. So it is implied that `daysLeftForNextPurchase` is greater than 7 :+1:  

You can see this in action by checking out this code snippet and changing the values of the `a` variable:
```
const a = 3;

if (a < 5) {
  console.log("first")
} else if (a < 8) {
  console.log("second")
} else {
  console.log("third")
}

Reply:
Thanks for the suggestion, I will change it :)